src/Commands/ToolsCommandGroup.cs

```diff
@@ -266,7 +266,8 @@ private static void AppendBanInformation(MemberData memberData, StringBuilder bu
     private async Task<Result> SendRandomNumberAsync(long first, long? secondNullable,
         IUser user, CancellationToken ct)
     {
-        var second = secondNullable ?? 0;
+        const int secondDefault = 0;
```

Comment:
we're working with longs
```suggestion
        const long secondDefault = 0;
```